packages/patternfly-4/react-integration/demo-app-ts/src/App.tsx

```diff
@@ -77,7 +74,7 @@ class App extends Component {
           action={<AlertActionLink>Action Button</AlertActionLink>}
         />
         <Alert variant="success" title="Success notification title" />
-        <Select
+        {/* <Select
```

Comment:
is select broken right now?

Reply:
I must have had outdated react-core in my node_modules folder so it couldn't find those exports. I cleared and reinstalled and now is working. I reverted the change to this file